docs/README.md

```diff
@@ -182,160 +284,142 @@ For example, if you wanted to query for information about `'U3 POWER CORP.'` you
+      "addresses": [],
+      "attributes": [ ... ],
       "credential_set": { ... },
       "credential_type": { ... },
-      "attributes": [ ... ],
-      "names": [ ... ],
-      "topic": {
-        "id": 787297,
-        "create_timestamp": "2020-09-17T19:09:38.681116-07:00",
-        "update_timestamp": "2020-09-17T19:09:38.681143-07:00",
-        "source_id": "BC0772006",
-        "type": "registration.registries.ca",
-        "names": [ ... ],
-        "local_name": { ... },
-        "remote_name": null,
-        "addresses": [],
-        "attributes": [ ... ]
-      },
-      "related_topics": []
-    },
-    ...
+      "inactive": false,
+      "revoked": false,
+      "effective_date": "2006-10-17T16:58:42-07:00",
+      "revoked_date": null
+    }
   ]
 }
 ```
 
-Each of the results returned is a foundational credential for the ___topic___ you are searching for. A foundational credential is generated when an organization is first registered in OrgBook. All other credentials for this organization are then linked (related to) this credential. It is possible to receive more than 1 foundational credential back from this query (especially for those belonging to related organizations). Results are returned in pages of up to 10 closely matching entities to the name you provide in the query. Typically you would iterate through the results and find the first credential where the `source_id` under the `topic` field matches the BC Registries number you passed in the name query.
+Each of the results returned is a foundational credential for the ___topic___ you are searching for. A foundational credential is generated when an organization is first registered in OrgBook. All other credentials for this organization are then linked (related to) this credential. It is possible to receive more than 1 foundational credential back from this query (especially for those belonging to related organizations). Results are returned in pages of up to 10 closely matching organizations to the query string you provide. Typically you would iterate through the results and find the first credential where the `source_id` matches the BC Registries number you passed as the query string.
 
 #### Step 2. Query for credentials
 
-Locate the `topic` field from the previous result and, within that, locate the `id` field. The topic id can be used to obtain all OrgBook credentials for an organization of interest.
-
-Make a `GET` request to the `/search/credential` endpoint, passing in the topic id from the topic search result to the `topic_id` query parameter in the request URL.
+Locate the `id` field from the previous result. The topic `id` can be used to obtain all credentials for an organization of interest from OrgBook.
 
-The same optional parameters for the `/search/topic` endpoint can be attached to the request.
+Make a `GET` request to the `/v4/topic/:id/credential-set` endpoint, substituting the `:id` URL parameter with the topic `id` from the previous topic search.
```

Comment:
Nit: maybe we could change the URL parameter to be in the swagger format to keep it consistent. `{id}` instead of `:id`